scss/tech.scss

```diff
@@ -0,0 +1,59 @@
+@import "colors";
```

Comment:
this should be in the application instead to avoid having to import it in every file that uses colors. also use single quotes for imports.